pex/bin/pex.py

```diff
@@ -457,9 +457,10 @@ def interpreter_from_options(options):
     return interpreter
 
 
-def build_pex(args, options, resolver_option_builder):
-  with TRACER.timed('Resolving interpreter', V=2):
-    interpreter = interpreter_from_options(options)
+def build_pex(args, options, resolver_option_builder, interpreter=None):
```

Comment:
apologies - this fell off my radar due to EOQ activity, but circling back now.

where might I find the `heron` command referenced in your comment above?